### ISSUE TITLE
Obtaining offsets to the bot(not really an issue)

### ISSUE THREAD

**Issue by MisterJacksen** [rohvani/CSGOBot#1]
If you could specify what you mean with CAMERA and LOCAL_CAMERA_POSITION. That'd be great! So I and other possible users know what to search for in olly.
I dont think those offsets are very common :8ball: 

Not really an issue, but I dont know how to contact you direcly.
Regards

**Comment by rohvani** [comment]
Hello!  Just saw this, I apologize for the late response.

LOCAL_CAMERA_POSITION is the world position of the camera, this is used for calculating the angle to aim towards a point.

CAMERA is the camera's current pitch and yaw.

Structure of these two addresses can be found in this file: https://github.com/Seefo/CSGOBot/blob/master/Botv2/Botv2/Utilities/Helpers/GameHelper.cs

If you need any further help, feel free to reply to this issue or contact me on Steam:  http://steamcommunity.com/id/Seefo

**Comment by MisterJacksen** [comment]
Haha, It's alright. 
It's just so that I'm kind of new in this scene and im trying to do some research about the source engine and pointers/multilevelpointers. I stumbled across your source and thought that I could move on from there.
Anyway, I replaced the most old offsets with new ones.

CAMERA is the  is the pitch and yaw. Isnt that usually the m_angRotation  offset? I tried using that offset. But it seems to be incorrect.
LOCAL_CAMERA_POSITION, isnt that m_vecOrigin+m_vecViewOffset? That does not work either.

 I'm sorry for overwhelming you with questions like this. I just need to know how you obtained the offsets for the two variables above.
